load-testing/tests/relays_stress_test.go

```diff
@@ -392,17 +388,13 @@ func (s *relaysSuite) MoreActorsAreStakedAsFollows(table gocuke.DataTable) {
 	// Parse and validate the actor increment plans from the given step table.
 	plans := s.parseActorLoadTestIncrementPlans(table)
```

Comment:
Why not just go with `s.plans := `?

Reply:
I believe the thinking was that they should be validated first but I suppose it's functionally equivalent if the test fails before or after the assignment.